API.md

```diff
@@ -197,6 +197,8 @@ Class | Method | HTTP request | Description
 *DevicesApi* | [**FindProjectDevices**](docs/DevicesApi.md#findprojectdevices) | **Get** /projects/{id}/devices | Retrieve all devices of a project
 *DevicesApi* | [**FindTraffic**](docs/DevicesApi.md#findtraffic) | **Get** /devices/{id}/traffic | Retrieve device traffic
 *DevicesApi* | [**GetBgpNeighborData**](docs/DevicesApi.md#getbgpneighbordata) | **Get** /devices/{id}/bgp/neighbors | Retrieve BGP neighbor data for this device
+*DevicesApi* | [**GetDeviceFirmwareSets**](docs/DevicesApi.md#getdevicefirmwaresets) | **Get** /devices/{id}/firmware-sets | Get Device&#39;s associated Firmware Set
```

Comment:
Adds support for device firmware sets (do we have a doc we could link to for this?)